doc/source/api.rst

```diff
@@ -62,3 +62,12 @@ Storage Pool
 
 .. autoclass:: pylxd.models.StoragePool
    :members:
+
+Cluster
+------------
```

Comment:
Sadly, RST is a bit picky, especially in my setup; the heading underline needs to be the same length as the title above it, please.